tests/datasets/test_landcoverai.py

```diff
@@ -44,13 +44,13 @@ def test_already_extracted(self, dataset: LandCoverAIGeo) -> None:
 
     def test_already_downloaded(self, monkeypatch: MonkeyPatch, tmp_path: Path) -> None:
         url = os.path.join("tests", "data", "landcoverai", "landcover.ai.v1.zip")
-        root = str(tmp_path)
+        root = tmp_path
         shutil.copy(url, root)
-        LandCoverAIGeo(root)
+        LandCoverAIGeo(str(root))
```

Comment:
Why does this need to be converted to a string?

Reply:
This was probably added earlier to fix mypy error when ``LandCoverAIGeo`` did not support ``pathlib.path``. Have fixed this and gone through other test files to fix similar issues.